packages/flutter/lib/src/animation/animations.dart

```diff
@@ -533,3 +533,66 @@ class TrainHoppingAnimation extends Animation<double>
     return '$currentTrain\u27A9$runtimeType(no next)';
   }
 }
+
+/// An interface for combining multiple Animations. Subclasses need only
+/// implement the `value` getter to control how the child animations are
+/// combined. Can be chained to combine more than 2 animations.
+/// Example: to create an animation that is the sum of two others, subclass
```

Comment:
blank line before the example.
introduce the example with "For example, to create...".
Put the "value = ..." stuff in back ticks.